src/Converter/Feature2Mesh.js

```diff
@@ -502,10 +477,23 @@ export default {
      * });
      */
     convert(options = {}) {
+        deprecatedFeature2MeshOptions(options);
         return function _convert(collection) {
             if (!collection) { return; }
 
-            return featuresToThree(collection.features, options);
+            const features = collection.features;
+
+            if (!features || features.length == 0) { return; }
+
+            const group = new THREE.Group();
```

Comment:
Here you create a `Group` without 'cleaning' the other previous groups.

Reply:
done